### PR TITLE
travis: Upload build artifacts for release branch.

### PR DESCRIPTION
**Pull request by priyank-p**
Only upload's artifacts for release branch.

### CODE REVIEW

File: scripts/prepare-artifacts.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+// go to dist directory
+process.chdir(path.resolve(__dirname, '../dist'));
+
+const files = fs.readdirSync(process.cwd());
+function changeFileName(file) {
+  file = file.replace(/^z/, 'Z');
+
+  if (/\.deb/.test(file)) {
+    file = file.replace(/[_]{1}/g, '');
```
Comment (akashnimare) [inline scripts/prepare-artifacts.js]: This should work -
```
file = file.replace(/_/g, '-')
```

**Comment by akashnimare** [comment]
@cPhost looks like we can't build the 32-bit installers, right?

**Comment by priyank-p** [comment]
Yeah, we can't yet, I will give a try sometime soon but yeah it will have to be with docker images.

**Comment by akashnimare** [comment]
@cPhost let's try the docker image.

**Comment by priyank-p** [comment]
Finally after some more tries, got something working properly... Opening a new PR.

**Pull request closed without merging**